.github/workflows/pr_lint.yml

```diff
@@ -6,8 +6,24 @@ on:
   pull_request:
     types: ['opened', 'edited', 'reopened', 'synchronize']
 jobs:
+  prologue:
+    name: prologue
+    runs-on: ubuntu-20.04
+    outputs:
+      job_skip: ${{ steps.prologue.outputs.job_skip }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: prologue
+      id: prologue
+      uses: ./.github/actions/ci_prologue
+      with:
+        GITHUB_EVENT_NAME: ${{ github.event_name }}
+        PR_COMMONS_BODY: "${{ github.event.pull_request.body }}"
+        GITHUB_WORKFLOW: ${{ github.workflow }}
```

Comment:
All PRs need to run this workflow.

Reply:
if like this, you can not specify run list or just specify this job to run.